README.md

```diff
@@ -212,6 +215,22 @@ Response:
 }
 ```
 
+### Filtering Subtitles

```

Comment:
The phrase "Filtering subtitles" is a bit ambiguous IMO, as it could mean "filtering individual subtitles" or "filtering subtitle text."